ecs-agent/tmds/handlers/fault/v1/types/types.go

```diff
@@ -96,6 +105,27 @@ func (request NetworkLatencyRequest) ValidateRequest() error {
 	if request.Sources == nil || len(request.Sources) == 0 {
 		return fmt.Errorf(missingRequiredFieldError, "Sources")
 	}
+	for _, element := range request.Sources {
+		elementStr := aws.StringValue(element)
+		validIp := true
+		if net.ParseIP(elementStr) == nil {
+			logger.Error("Found invalid IP", logger.Fields{
+				field.Error: fmt.Errorf(invalidValueError, elementStr, "Sources"),
+			})
```

Comment:
If the caller has provided only a CIDR block then this error log will be printed although there is no real error. I think we should avoid doing that.

Reply:
that's a good point

Reply:
Removed both logs because if any value is invalid, we will 1. log the response for debugging and 2. return the explicit msg to clients.